INSTALL.md

```diff
@@ -61,10 +61,15 @@ The code also requires a working version of glue, supporting `glue.ligolw.ligolw
 The code uses cupy to access GPUs.  If you don't have one, the code will still work.
 If you do need one, make sure to install cupy **on a machine that supports GPUs **
 
+You probably do not want to pip install `cupy` directly, but rather a pre-compiled version for your system's CUDA version.  To find your system's version (assuming it's installed) run `nvidia-smi`, and look for "CUDA Version" in the top-right.  For version X.Y, the appropriate cupy package will be called `cupy-cudaXY` (e.g., if you have CUDA 11.0, install `cupy-cuda110`)
```

Comment:
This is strictly speaking not accurate. `CUDA Version` here refers to *the maximal CUDA Toolkit version supported by the driver version reported on its left*, but what you really need is the runtime library version installed locally, which can be decoupled/fall behind this version (say you have CUDA 10.2 installed, but you upgrade the driver to 11.1). Perhaps it's better to use `nvcc --version` if there is a local installation available to check.

FYI in some environments where there is no CUDA Toolkit installed locally, but has the driver installed, you can install CuPy from Conda-Forge:
```
conda install -c conda-forge cupy cudatoolkit=X.Y
```
`X.Y` is again up to the driver version. The `cudatoolkit` contains all CUDA runtime libraries (but no `nvcc` or the compiler toolchain).